app/controllers/application_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
```

Comment:
- [OPTIONAL ] Splash screen. According to the project requirements, The app should have a simple page with the name of the student's app with links to the sign-up and log-in pages. Kindly add this 👍🏻 
- This is only optional because due to oversight the previous reviewer probably missed them. However, I strongly remand you do it 👍🏻 

![Screenshot from 2022-10-30 03-28-35](https://user-images.githubusercontent.com/79058364/198858533-b90a93a1-162b-4838-ad23-ca6385571651.png)